databases/week2/PhoneStore Database.sql

```diff
@@ -0,0 +1,35 @@
+CREATE DATABASE phoneStore;
+
+USE phoneStore;
+
+CREATE TABLE phones (
+    `phone_id` INT PRIMARY KEY,
```

Comment:
This can be just 'id' since it is a primary key not to get confused it with 'phone_id' from 'phones_orders' table. Same applies to any other primary keys (id's). ☺️

Reply:
But isn´t just "id" a little bit confusing as well? How about I change the "phone_id" from the phones_orders for "phones_orders_id"?